examples/solverdummies/precice-config.xml

```diff
@@ -10,40 +10,40 @@
   </log>
 
   <solver-interface dimensions="3">
-    <data:vector name="dataOne" />
-    <data:vector name="dataTwo" />
+    <data:vector name="Data-One" />
+    <data:vector name="Data-Two" />
 
-    <mesh name="MeshOne">
-      <use-data name="dataOne" />
-      <use-data name="dataTwo" />
+    <mesh name="Mesh-One">
```

Comment:
```suggestion
    <mesh name="SolverOne-Mesh">
```

as stated in the [conventions](https://precice.org/community-contribute-to-precice.html#naming-conventions):

> Mesh names start with the participant/domain name, e.g. Fluid-Mesh

Reply:
re modified the mesh names to include the solver name in #[2301cb4](https://github.com/precice/precice/pull/1548/commits/2301cb4c9ce6ade0b84562ba92b3b6c432c61ac6)